graylog2-web-interface/src/views/components/views/ViewPropertiesModal.jsx

```diff
@@ -23,7 +24,8 @@ export default class ViewPropertiesModal extends React.Component {
     };
   }
 
-  componentWillReceiveProps(nextProps) {
+  // eslint-disable-next-line camelcase
```

Comment:
I does not have to be in this PR, but wouldn't it be wiser to make a functional component of this component, instead of adding a UNSAFE_